_i18n/en/resources/user-guides/mine-to-pool.md

```diff
@@ -42,7 +42,7 @@ this file, extract the archive somewhere memorable, like your desktop.
 
 ### Other distributions and operative systems
 
-Binaries are available for multiple distributions and operative systems, but you can also build the miner by yourself. Take a loot at [XMrig's docs](https://xmrig.com/docs/miner)
+Binaries are available for multiple distributions and operative systems, but you can also build the miner by yourself. Take a look at [XMrig's docs](https://xmrig.com/docs/miner)
```

Comment:
```suggestion
Binaries are available for various distributions and operating systems, but you can also build the miner by yourself. Take a look at [XMrig's docs](https://xmrig.com/docs/miner)
```

While `various` is simply my personal preference
`operating` vs `operative` looks like a spelling mistake